gulpfile.js

```diff
@@ -241,9 +241,9 @@ gulp.task('run-test-integration', function() {
 });
 
 gulp.task('test-integration', function(cb) {
-  return runSequence(
-    'start-simulators', 'run-test-integration', 'stop-simulators', cb
-  );
+  runSequence('start-simulators', 'run-test-integration', function tearDown() {
```

Comment:
I know, the function can remain anonymous, but I find the intent of wrapping the callback clearer with that kind of names